crates/gitql-ast/src/function.rs

```diff
@@ -242,6 +250,22 @@ fn text_char(inputs: Vec<Value>) -> Value {
     return Value::Text("".to_string());
 }
 
+fn text_right(inputs: Vec<Value>) -> Value {
+    let text = inputs[0].as_text();
+    if text.is_empty() {
+        return Value::Text("".to_string());
+    }
+
+    let number_of_chars = inputs[1].as_number() as usize;
+    if number_of_chars > text.len() {
+        return Value::Text(text);
+    }
+
+    let text = text.as_str();
```

Comment:
You can write 

```rs
return Value::Text(text[text.len() - number_of_chars..text.len()].to_string());
``` 

directly without need to 

```rs
let text = text.as_str();
```

Can you please create new variable to store the start to make it more readable

```rs
let start = text.len() - number_of_chars;
return Value::Text(text[start..text.len()].to_string());
```